### PR TITLE
[167] User creation process enhancement

### PR DESCRIPTION
**Pull request by mnashat**
- added different styled scroll bar for study room list
- redirected user to calendar after new user creation instead of redirecting to the login screen
- fixed a bug where entering incorrect login credentials reloaded the login page instead of displaying the "incorrect username/password" error message
- modified study room creation code to include study room owner as the first user in the participants list

### CODE REVIEW

**Review by EyalAzimov — APPROVED** [review]

**Comment by kikkiii21** [comment]
When I create an account I see the calendar but immediately logs me out. That means it's a cookie thing? Everytime we create an account we need to clear cookies?

**Comment by mnashat** [comment]
> When I create an account I see the calendar but immediately logs me out. That means it's a cookie thing? Everytime we create an account we need to clear cookies?

No. This branch works along with [164_new_user_process_enhancement](https://github.com/guaj/SemesterPlanner-Server/tree/164_new_user_process_enhancement) in the backend. Please make sure you are using both branches together. Thanks!

**Comment by kikkiii21** [comment]
I had the wrong BE branch! It works now

File: src/components/StudyRoom/StudyRoomCreate.js [hunk 1]
```diff
@@ -50,7 +49,7 @@ export default function RoomCreation() {
     const handleRoomCreation = (e) =>{
         e.preventDefault();
         //add the owner to the list
-        roomData.participants.push(roomData.owner);
+        // roomData.participants.push(roomData.owner);
```
Comment (kikkiii21) [inline src/components/StudyRoom/StudyRoomCreate.js:52]: please remove commented code

**Review by kikkiii21 — APPROVED** [review]
Great job!

File: src/components/StudyRoom/StudyRoomCreate.js (hunk 1 above)
Comment (mnashat) [reply src/components/StudyRoom/StudyRoomCreate.js]: Removed. Thanks for pointing it out!

**Pull request merged**